firstvoices/backend/serializers/join_request_serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from backend.models.join_request import JoinRequest
+from backend.serializers.base_serializers import (
+    BaseSiteContentSerializer,
+    UserDetailFieldMixin,
+    base_timestamp_fields,
+)
+from backend.serializers.fields import SiteHyperlinkedIdentityField
+
+
+class JoinRequestDetailSerializer(UserDetailFieldMixin, BaseSiteContentSerializer):
+    url = SiteHyperlinkedIdentityField(view_name="api:join-request-detail")
+    status = serializers.CharField(source="get_status_display")
+    reason = serializers.CharField(source="get_reason_display")
```

Comment:
Adding these get_x_display methods will work for reading, but it creates a bit of work for the writing side of the API. Have a look at this EnumField I started in a branch-- might work for both of these: https://github.com/First-Peoples-Cultural-Council/fv-be/commit/115a1d052ad244f87fc5e404572a6171b3c5e325#diff-707267488b2c19708dd8e499033508841531b948f395ee7e18b0036a2596ea16